addons/payment_authorize/models/authorize_request.py

```diff
@@ -58,9 +62,17 @@ def _authorize_request(self, data):
 
         :param etree._Element data: etree data to process
         """
+        logged_data = data
         data = etree.tostring(data, encoding='utf-8')
+
+        for node in logged_data.xpath('//merchantAuthentication'):
+            node.getparent().remove(node)
+        logged_data = etree.tostring(logged_data, encoding='utf-8')
+
+        _logger.info('_authorize_request: Sending values to URL %s, values:\n%s' % (self.url, pprint.pformat(logged_data)))
```

Comment:
`logged_data` is a string here, so is this really going to be readable? Or is everything going to be printed on one line?

Reply:
etree.tostring(x, pretty_print=True)